themes/bootstrap3/templates/search/results.phtml

```diff
@@ -121,10 +121,10 @@
 
 <?php /* Refine Search Options */ ?>
 <?php if ($recommendations): ?>
-  <div class="<?=$this->layoutClass('sidebar')?>" id="search-sidebar">
+  <aside class="<?=$this->layoutClass('sidebar')?>" id="search-sidebar">
```

Comment:
I feel like the idea of turning the recommendation sidebar into an aside has been previously discussed and rejected, since the sidebar usually presents core functionality like faceting, which is more than (to quote the definition of aside) "only indirectly related to the document's main content." But maybe I'm misremembering. Maybe @crhallberg remembers this better than I do.

Reply:
`region` will be more appropriate with the right aria-label. I could have sworn there was a `h2` denoting the search filters...

Reply:
There is no h2 specifically above the region where recommendations are placed; instead, each recommendation block within that region has its own h2. In most configurations, [the one in sidefacets.phtml](https://github.com/vufind-org/vufind/blob/dev/themes/bootstrap3/templates/Recommend/SideFacets.phtml#L36) ends up on top.